run/add_ddos_protection_iptables_rule.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-# Author: Wade Barnes https://gist.github.com/WadeBarnes/
-# Minor modifications: Sebastian Schmittner https://github.com/Echsecutor
+# Original author: Wade Barnes https://gist.github.com/WadeBarnes/
```

Comment:
@Echsecutor  Do we have a direct link to gist since its public not searchable ?  If not it's also fine.